packages/dns-discovery/src/index.ts

```diff
@@ -15,6 +15,37 @@ export { NodeCapabilityCount };
 
 const log = debug("waku:peer-discovery-dns");
 
+const enrTree = `enrtree://AOGECG2SPND25EEFMAJ5WF3KSGJNSGV356DSTL2YVLLZWIV6SAYBM@test.waku.nodes.status.im`;
```

Comment:
Sorry missed that earlier. Two changes:
1. Have both values for wakuv2.prod and wakuv2.test fleet available
2. Use wakuv2.prod fleet value by default (not test)